### ISSUE TITLE
setWebRtcBasedAcousticEchoCanceler

### ISSUE THREAD

**Issue by sm2017** [cloudwebrtc/flutter-webrtc#738]
Newer versions of WebRTC has `WebRtcAudioUtils` class which you can use for WebRTC based Acoustic echo cancellation (AEC).

```java
WebRtcAudioUtils.setWebRtcBasedAcousticEchoCanceler(true);
```

@cloudwebrtc Is there similar option on `flutter-webrtc` ?

**Comment by janhelwich** [comment]
Hi, I have the same question as we experience some troubles with echo. @sm2017 when you say "new versions of " does that mean you know the SDK used by flutter_webrtc does not have the `WebRtcAudioUtils`? 

@cloudwebrtc : Not sure if i get it right, but having [a look in here](https://chromium.googlesource.com/external/webrtc/+/HEAD/sdk/android/api/org/webrtc/PeerConnectionFactory.java) makes me guess, that the mentioned PR would not give the ability we are looking for.

**Comment by sm2017** [comment]
@janhelwich No, I mean there is no way to call `WebRtcAudioUtils` methods from dart

**Comment by janhelwich** [comment]
@sm2017 what is the problem you want to solve with this and did find another way to solve it maybe.

@cloudwebrtc do u still think it would be possible to set this via the mentioned PR and if so how?

For us the problem is that we experience strong echoes in some situations (absolutely unclear when and when not). So it would be good to at least try this setting.

**Comment by sm2017** [comment]
@janhelwich my searches shows that this method can help me to reduce echo

**Comment by tigrenok00** [comment]
Was this implemented in the release?

**Comment by taipoxin** [comment]
Are there any updates on it?
